meteortracker/save_event.py

```diff
@@ -0,0 +1,126 @@
+"""
+@author(s): Nathan Heidt, Jean Nassar
+
+In charge of saving meteor events to either a remote server or on the local
+machine for debugging or future uploading.
+
+It will upload to local and/or remote databases depending on what is specified
+in the config.ini file.
+
+"""
+import configparser
+import datetime
+import numbers
+import sqlite3
+import sys
+
+import cv2
+
+
+def _read(variable):
+    return getattr(sys.modules[__name__], variable)
+
+
+class EventLogger(object):
+    def __init__(self, config_path='config.ini'):
+        self.config = configparser.ConfigParser()
+        self.config.read(config_path)
+
+        #get working variables from config file
+        self.local_db_name = self.config.get('Database', 'Local', 0)
+        self.local_db_table_name = self.config.get('Database', 'LocalTable', 0)
+        self.remote_db_name = self.config.get('Database', 'Remote', 0)
+        self.local_image_location = self.config.get('Database',
+                                                    'LocalImages',
+                                                    0)
+
+        self.use_local = bool(self.config.get('Database', 'UseLocal', 0))
+        self.use_remote = bool(self.config.get('Database', 'UseRemote', 0))
+
+        self.conn = sqlite3.connect(self.local_db_name)
+        # connect to db and create tables if they don't exist
+        self.check_local_db()
+
+        self._variables = ['current_image', 'previous_image', 'date',
```

Comment:
this must be instantiated before it is used in check_local_db().  This is a better approach then the huge string that was in use before.